packages/client/src/runtime/utils/serializeRawParameters.ts

```diff
@@ -48,6 +48,7 @@ function encodeParameter(parameter: any, objectSerialization: 'fast' | 'slow'):
   if (isArrayBufferLike(parameter) || ArrayBuffer.isView(parameter)) {
     return {
       prisma__type: 'bytes',
+      // TODO Can we change this now in v16?
```

Comment:
Does not seem so, `ArrayBufferView` is still not accepted in `from`. Maybe we should send a PR.